### PR TITLE
Extending 'ca-certificates-base' by 2 new certificates.

### PR DESCRIPTION
**Pull request by PawelWMS**
###### Merge Checklist  

**All** boxes should be checked before merging the PR *(just tick any boxes which don't apply to this PR)*
- [x] The toolchain has been rebuilt successfully (or no changes were made to it)
- [x] The toolchain/worker package manifests are up-to-date
- [x] Any updated packages successfully build (or no packages were changed)
- [x] All package sources are available
- [x] cgmanifest files are up-to-date and sorted (`./cgmanifest.json`, `./toolkit/tools/cgmanifest.json`, `./toolkit/scripts/toolchain/cgmanifest.json`)
- [x] All source files have up-to-date hashes in the `*.signatures.json` files
- [x] `sudo make go-tidy-all` and `sudo make go-test-coverage` pass
- [x] Documentation has been updated to match any changes to the build system
- [x] Ready to merge

---

###### Summary 

Some of the applications supported in CBL-Mariner require a secure connection to a server, which got its certificate signed by a Microsoft-trusted CA (Certificate Authority). So far `ca-certificates-base` contained 2 sets of Microsoft-owned trusted anchors:

- revoked "Microsoft IT TLS CA 2" - needed until our package repository gets a new certificate signed by one of the CAs from the point below;
- valid "Microsoft Azure TLS Issuing CA 0*" certs - Microsoft-owned CAs currently recommended for issuing new certificates.

However, that not all services and server (for instance Azure Watson) can switch to certificates signed by a CA from the second set and still need to hold on to certificates issued by a CA from a (valid) third set, which is the one added in this PR.

###### Change Log  

- Added "Microsoft RSA TLS CA 01" and "Microsoft RSA TLS CA 02" to trusted anchors installed with `ca-certificates-base`.

###### Does this affect the toolchain?  

No.

###### Test Methodology

- Local build.
- Verified DNF and TDNF still work correctly after running a VHD image containing the new certificates.

### CODE REVIEW

File: SPECS/ca-certificates/certdata.base.txt [hunk 1]
```diff
@@ -942,3 +942,331 @@ CKA_TRUST_SERVER_AUTH CK_TRUST CKT_NSS_TRUSTED_DELEGATOR
 CKA_TRUST_EMAIL_PROTECTION CK_TRUST CKT_NSS_MUST_VERIFY_TRUST
 CKA_TRUST_CODE_SIGNING CK_TRUST CKT_NSS_MUST_VERIFY_TRUST
 CKA_TRUST_STEP_UP_APPROVED CK_BBOOL CK_FALSE
+
+#
+# Certificate "Microsoft RSA TLS CA 01"
+#
+# Issuer: CN=Baltimore CyberTrust Root,OU=CyberTrust,O=Baltimore,C=IE
```
Comment (jslobodzian) [inline SPECS/ca-certificates/certdata.base.txt:949]: >CN=Baltimore CyberTrust Root,O [](start = 10, length = 30)

I assume these are not related to the other baltimore certs?

Comment (PawelWMS) [reply SPECS/ca-certificates/certdata.base.txt:949]: I'm not sure what you mean, I'll summarize what this is:

1. This intermediate CA cert **is** signed by the Baltimore root CA. However, we're **not** trusting the root Baltimore CA itself with this change, we're trusting only these 2 new intermediate CAs. These CAs are added to the `ca-certificates-base` package btw.
2. There currently are no Microsoft-owned root CAs we could trust, that's why `ca-certificates-base` consists of a few intermediate CA certs.
3. The Baltimore root CA (along with all certs it has issued) is trusted if you install the Mozilla bundle provided by the `ca-certificates` package but that's not part of the core packages, only `ca-certificates-base` is there.

**Review by jslobodzian — CHANGES_REQUESTED** [review]
:clock1:

**Review by jslobodzian — APPROVED** [review]
:shipit:
